frontend/src/export/mtgo.spec.js

```diff
@@ -0,0 +1,43 @@
+const {describe, it} = require("mocha");
+const assert = require("assert").strict;
+
+import { exampleDeck as deck } from "./test-cards.js";
+import mtgo from "./mtgo.js";
+
+describe("export/mtgo", () => {
+  // console.log(multiFacettedCards.map(c => ([c.name, c.layout])))
```

Comment:
Maybe deleting these artifacts could be nice ?

Reply:
Agree. Thanks will do